spanner/opentelemetry/tracing/Readme.md

```diff
@@ -0,0 +1,49 @@
+# Cloud Spanner OpenTelemetry Traces
+
+## Setup
+
+This sample requires [Go](https://go.dev/doc/install).
+
```

Comment:
I think it would be useful to say here "Ensure that your Go runtime version is supported by the OpenTelemetry-Go compatibility policy before enabling OpenTelemetry instrumentation. Refer to compatibility here https://github.com/googleapis/google-cloud-go/blob/main/debug.md#opentelemetry"

Reply:
Also, this sample requires `gcloud`

Reply:
Done